### ISSUE TITLE
Color palette

### ISSUE THREAD

**Issue by JonathanRufino** [NotifiCar/NotifiCar#12]
A consistent color palette is needed

**Comment by andeladenaro** [comment]
What about change the color scheme to red, red will represent attention and urgency.

**Comment by JonathanRufino** [comment]
Yeah, I thought about changing the color scheme to match the icon you proposed.

Thank you 😃

**Comment by luishng** [comment]
I applied the suggested color palette for the application:

<img src="https://user-images.githubusercontent.com/9291016/38380073-092e9532-38d9-11e8-88d4-f5bc89d48c31.png" width="300"/><img src="https://user-images.githubusercontent.com/9291016/38380074-09549610-38d9-11e8-8926-d80a092f2f6c.png" width="300"/>
<img src="https://user-images.githubusercontent.com/9291016/38380076-09788cdc-38d9-11e8-90b1-3ab05e09f13e.png" width="300"/><img src="https://user-images.githubusercontent.com/9291016/38380077-099ce5fa-38d9-11e8-97e5-efff5af5532c.png" width="300"/>

The last image will be changed by a splash screen with the new application icon instead of this current screen. I think the colors taking a ScreenShot did not look the same as on the device, I liked it as they did.

If @andeladenaro have some more suggestions of UI/UX to apply it would be great.
